pkgs/tools/security/osv-scanner/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  osv-scanner,
```

you will be asked to use `nixpkgs-fmt`:
```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
, testers
, osv-scanner
```